### PR TITLE
Preserve Ruby 2.7 behavior for Ruby keyword args

### PR DESCRIPTION
**Pull request by Jonic**
Applies [`ruby2_keywords`](https://github.com/ruby/ruby2_keywords) to the `Command.call` method, which preserves the Ruby 2.7 behavior for keyword args.

The result of these changes is that Rectify will continue to work with the existing behavior for keyword args once we've upgraded Panacea to Ruby 3. We've had to take this approach because fixing it would have also required us to upgrade Rectify to only work on Ruby `>= 3.0`. This is due to the fact that our particular use of kwargs in the Command pattern are considered a "corner case", preventing us from adopting the new behavior in our Ruby 2.7 code.

See: https://www.ruby-lang.org/en/news/2019/12/12/separation-of-positional-and-keyword-arguments-in-ruby-3-0/ - search this page for the string "If your code doesn’t have to run on Ruby 2.6 or older"

## So what does all this mean?

In short, we don't have to worry about Rectify anymore. Once this has merged, we've released 0.14.0, and we've updated Panacea to use this new version, our existing use of Rectify Commands will continue to work even after we've upgraded the application to run on Ruby 3.0.

This allows us to almost forget about the old Rectify-powered commands, and begin thinking about what tool or pattern we introduce to replace Rectify.

### CODE REVIEW

File: spec/fixtures/command/args_command.rb [hunk 1]
```diff
@@ -1,3 +0,0 @@
-class ArgsCommand < Rectify::Command
-  def initialize(_, _, _); end # rubocop:disable Lint/MissingSuper, Style/RedundantInitialize
-end
```
Comment (Jonic) [inline spec/fixtures/command/args_command.rb:3]: This file was replaced by `spec/fixtures/commands/args_command.rb`, below. It's not shown up as a diff of the same file because the directory was also renamed for better use of Guard.

**Review by ronanduddy — CHANGES_REQUESTED** [review]
Nice one 👍, can we bump to `0.14.1`?

**Comment by Jonic** [comment]
> Nice one 👍, can we bump to `0.14.1`?

@ronanduddy No need to. We haven't released 0.14.0 yet. Just because code is in `master` doesn't necessarily mean it's released. It's quite normal for dependencies to treat `master` as the `next` version, so we can add things here until we're ready to release 0.14.0, at which point we will [create a tag for the new release](https://github.com/healiosuk/rectify/tags), and then we can install it in Panacea's Gemfile.

So basically once this merges, 0.14.0 will be done. I'll get the tag created, and then open a PR to install 0.14.0 on Panacea.

**Review by ronanduddy — APPROVED** [review]
Ah got you @Jonic - I just checked Panacea's Gemfile:
```ruby
gem "rectify", "~> 0.13", github: "healiosuk/rectify", tag: "v0.13.0"
```
LGTM
